### PR TITLE
KEA-156 setup kumbh sans and noto sans

### PR DESCRIPTION
**Pull request by hyunghoseo**

### CODE REVIEW

File: frontend/src/App.tsx [hunk 1]
```diff
@@ -1,11 +1,37 @@
...
+import { useFonts } from "expo-font";
+import * as SplashScreen from "expo-splash-screen";
+import { useCallback } from "react";
 import { View, StyleSheet } from "react-native";
 
 import Header from "./components/Header/Header";
 import AppNavigator from "./navigation/AppNavigator";
 import { linking } from "./navigation/navigationLinks";
 
+SplashScreen.preventAutoHideAsync();
```
Comment (calvinyoon) [inline frontend/src/App.tsx:16]: what does this do?

File: frontend/src/App.tsx [hunk 2]
```diff
@@ -1,11 +1,37 @@
...
+
 const App = () => {
+  const [fontsLoaded, fontError] = useFonts({
```
Comment (calvinyoon) [inline frontend/src/App.tsx:19]: how much of this (lines 19-34) was from documentation vs coded from scratch?

Comment (hyunghoseo) [reply frontend/src/App.tsx:19]: everything was taken from this documentation https://docs.expo.dev/develop/user-interface/fonts/

File: frontend/src/App.tsx (hunk 1 above)
Comment (hyunghoseo) [reply frontend/src/App.tsx:16]: refer to the other comment

File: frontend/src/App.tsx (hunk 2 above)
Comment (calvinyoon) [reply frontend/src/App.tsx:19]: thanks!

**Review by calvinyoon — DISMISSED** [review]

File: frontend/src/App.tsx (hunk 1 above)
Comment (doolri1276) [reply frontend/src/App.tsx:16]: @calvinyoon resolve your comments before approving

File: frontend/src/components/Header/Header.tsx [hunk 3]
```diff
@@ -202,8 +202,8 @@ const getStyles = (headerHeight: number) =>
       padding: 8,
     },
     navText: {
+      fontFamily: "KumbhSans_500Medium",
```
Comment (doolri1276) [inline frontend/src/components/Header/Header.tsx:205]: Could you update the font color to match with Figma? 2A2A2A

File: frontend/src/components/Header/Header.tsx [hunk 4]
```diff
@@ -246,8 +246,8 @@ const getStyles = (headerHeight: number) =>
       paddingVertical: 25,
     },
     menuText: {
+      fontFamily: "KumbhSans_600SemiBold",
```
Comment (doolri1276) [inline frontend/src/components/Header/Header.tsx:249]: Same here

File: frontend/package.json
Comment (doolri1276) [inline frontend/package.json:1]: Please rebase.

**Review by doolri1276 — CHANGES_REQUESTED** [review]

**Review by doolri1276 — APPROVED** [review]

**Comment by doolri1276** [comment]
Approved

**Pull request merged**